perf-tests/src/test/scala/AkkaHttpTapirSimulation.scala

```diff
@@ -0,0 +1,9 @@
+package perfTests
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+class AkkaHttpTapirSimulation extends Simulation {
+  val httpProtocol = http.baseUrl("http://127.0.0.1:8080/akka-http-tapir")
+  setUp(perfTests.Common.scn.inject(atOnceUsers(1000)).protocols(httpProtocol))
```

Comment:
it might be interested to compare, what happens when there's much less (e.g. 100) concurrent users - how does the throughput in req/s change for vanilla akka-http and tapir?